pubspec.yaml

```diff
@@ -49,6 +49,7 @@ dev_dependencies:
 
   flutter_test:
     sdk: flutter
+  test:
```

Comment:
why?
this is not required

Reply:
Apologies about that. Changes have been made and pushed. Kindly check